EOLib/Domain/Login/PlayerInfoRepository.cs

```diff
@@ -53,6 +57,8 @@ public sealed class PlayerInfoRepository : IPlayerInfoRepository, IPlayerInfoPro
 
         public bool PlayerHasAdminCharacter { get; set; }
 
+        public bool IsPlayerFrozen { get; set; }
```

Comment:
`PlayerInfoRepository` and interfaces are largely intended to be a pre-game menu thing. Once the player is in game, `Character` should be used instead.

`Frozen` state should be moved to the `Character` class in EOLib. I think this would also eliminate a lot of churn in the diffs due to introducing the `IPlayerInfoProvider` dependency where there's already a `CharacterProvider`.